diff --git a/node_modules/ytdl-core/lib/info.js b/node_modules/ytdl-core/lib/info.js
index 417f876..f361f84 100644
--- a/node_modules/ytdl-core/lib/info.js
+++ b/node_modules/ytdl-core/lib/info.js
@@ -60,8 +60,8 @@ exports.getBasicInfo = async(id, options) => {
   };
   let info = await pipeline([id, options], validate, retryOptions, [
     getWatchHTMLPage,
-    getWatchJSONPage,
-    getVideoInfoPage,
+    //getWatchJSONPage,
+    //getVideoInfoPage,
   ]);
 
   Object.assign(info, {
@@ -251,7 +251,7 @@ const parseJSON = (source, varName, json) => {
 
 
 const findJSON = (source, varName, body, left, right, prependJSON) => {
-  let jsonStr = utils.between(body, left, right);
+  let jsonStr = body.split("ytInitialPlayerResponse = {")[1].split("}}};")[0] += "}}}";
   if (!jsonStr) {
     throw Error(`Could not find ${varName} in ${source}`);
   }
